receiver/couchdbreceiver/metadata.yaml

```diff
@@ -0,0 +1 @@
+name: couchdb
```

Comment:
```suggestion
name: couchdb

```

Reply:
added new line. I believe this should be couchdbreceiver right?